extensions/wikia/Listusers/SpecialListusers.php

```diff
@@ -12,54 +12,60 @@
 	exit( 1 ) ;
 }
 
-$wgExtensionCredits['specialpage'][] = array(
+$GLOBALS['wgExtensionCredits']['specialpage'][] = [
```

Comment:
Is this some new MW convention? If so, how should we refer to global variables in methods? Should we keep using `global` keyword or `$GLOBALS`? The latter one looks cleaner and self explanatory, especially in long methods.

Reply:
Maybe [this](https://github.com/Wikia/app/pull/14341#discussion_r154668912) is related.